setup.py

```diff
@@ -20,5 +20,6 @@
       packages=['fluent', 'fluent.syntax', 'fluent.migrate'],
       package_data={
           'fluent.migrate': ['cldr_data/*']
-      }
-      )
+      },
+      install_requires=['six']
```

Comment:
Do we actually need this if it's only used for testing?

Reply:
You can use tests_require, I think.